### ISSUE TITLE
Browser Bundle

### ISSUE THREAD

**Issue by SilentRhetoric** [algorandfoundation/algokit-utils-ts#119]
Similarly to how algosdk has a minified browser bundle, could one be created for this library so that it can be imported from CDN in the browser?

[https://github.com/algorand/js-algorand-sdk#browser](https://github.com/algorand/js-algorand-sdk#browser)

**Comment by robdmoore** [comment]
Makes sense, we've added to the enhancements backlog for prioritisation.
